src/Standards/PEAR/Tests/Commenting/FunctionCommentUnitTest.inc

```diff
@@ -9,7 +9,7 @@ class PHP_CodeSniffer_File
      *
      * @param int $stackPtr   The position in @ @unknown the stack of the token
      *                        that opened the scope
-     * @param int $detph    How many scope levels down we are.
```

Comment:
This change should be reverted as this is actually testing something.

Reply:
Yeh, I tried to deduce which were "accidental" typos, and which ones were actually testing sniffs.

Reply:
fixed